src/wide/u32x8_t.rs

```diff
@@ -71,11 +97,75 @@ impl u32x8 {
                 Self(cmp_eq_mask_i32_m128i(self.0, rhs.0), cmp_eq_mask_i32_m128i(self.1, rhs.1))
             } else if #[cfg(all(feature = "simd", target_feature = "simd128"))] {
                 Self(u32x4_eq(self.0, rhs.0), u32x4_eq(self.1, rhs.1))
+            } else if #[cfg(all(feature = "simd", target_arch = "aarch64", target_feature = "neon"))] {
```

Comment:
aarch64 can be without neon?

Reply:
Ohh, I'm not entirely sure tbh. You can certainly tell Rust to compile for aarch64 without that target feature though, which would start breaking things. So you might say "alright let's just remove the target_arch check then instead", but there's Neon on ARMv7 too and it doesn't some of these instructions (I may look into adding support for that before merging too).

Reply:
Well, Rust crashes when a user tries to disable sse/sse2, so... If you think it's needed, then fine. As long as it runs on M1 I'm fine.